SWLOR.Game.Server/Feature/AbilityDefinition/Force/BenevolenceAbilityDefinition.cs

```diff
@@ -28,13 +28,27 @@ private void Impact(uint activator, uint target, int baseAmount)
             var targetBonus = willBonus;
             if (target != activator && Stat.GetCurrentFP(activator) >= 16)
             {
-                var willRestore = (willBonus / 2);
+                var willRestore = (willBonus / 2) * 4;
+                var duration = (willBonus * 60f);
```

Comment:
You need to set a default duration. If the player has a negative mod, this will immediately expire. I would recommend adding 90f seconds to this to be safe.